styles/base/_typography.scss

```diff
@@ -8,79 +8,61 @@ body {
 
 .hero {
   &__tag {
+    @include font-overline;
+
     padding: 0;
     margin: 0;
-
-    &--home {
-      @include font-overline;
-
-      color: var(--pueblo);
-    }
-
-    &--child {
-      @include font-overline;
-
-      color: var(--pueblo);
-    }
+    color: var(--pueblo);
   }
 
   &__title {
+    @include font-h3;
+
     padding: 0;
     margin: 0;
+    color: var(--tuscany);
 
-    &--home {
-      @include font-h3;
+    &--profile {
+      font-weight: 300;
 
-      color: var(--tuscany);
-    }
-
-    &--child {
-      @include font-h3;
-
-      color: var(--tuscany);
+      b {
+        background-color: var(--pink);
+      }
     }
   }
 
   &__body {
+    @include font-subtitle1;
+
     padding: 0;
     margin: 0;
+    color: var(--cod-gray);
 
-    &--home {
-      @include font-subtitle1;
-
-      color: var(--cod-gray);
-
-      b {
-        background-color: var(--pink);
-        font-weight: 500;
-      }
+    &--profile {
+      @include font-body1;
     }
 
-    &--child {
-      @include font-subtitle1;
-
-      color: var(--cod-gray);
-
-      b {
-        background-color: var(--pink);
-        font-weight: 500;
-      }
+    b {
+      background-color: var(--pink);
+      font-weight: 500;
     }
   }
 }
 
 @media screen and (min-width: $tablet-viewport) {
```

Comment:
$tablet-viewport have 768px then if you use mobile-first is necessary write +1 to have the correct viewport on tablet

Reply:
Thank you Adrian, will get right to it soon.